xilem/examples/http_cats.rs

```diff
@@ -197,8 +199,8 @@ impl Status {
             FlexSpacer::Fixed(10.),
             image,
             // TODO: Overlay on top of the image?
-            // HACK: Trailing spaces workaround scrollbar covering content
-            prose("Copyright ©️ https://http.cat      ").alignment(TextAlignment::End),
+            sized_box(prose("Copyright ©️ https://http.cat").alignment(TextAlignment::End))
+                .padding(Padding::trailing(20.)),
```

Comment:
We should still document this as a hack to avoid overlap with the scrollbar, but it is certainly a better solution.

Reply:
Sure. I added the comment back in.